go/libkb/assertion.go

```diff
@@ -281,6 +288,8 @@ func (a AssertionHTTP) CollectUrls(v []AssertionURL) []AssertionURL        { ret
 func (a AssertionHTTPS) CollectUrls(v []AssertionURL) []AssertionURL       { return append(v, a) }
 func (a AssertionDNS) CollectUrls(v []AssertionURL) []AssertionURL         { return append(v, a) }
 func (a AssertionFingerprint) CollectUrls(v []AssertionURL) []AssertionURL { return append(v, a) }
+func (a AssertionPhoneNumber) CollectUrls(v []AssertionURL) []AssertionURL { return append(v, a) }
+func (a AssertionPhoneNumber) IsServerTrust() bool                         { return true }
```

Comment:
seems like this should cascade up and infect AND and ORs

Reply:
oh this is a method on AssertionURL, not AssertionExpression as I had thought. nevermind

Reply:
oh ok, I forgot to ask you about that. thanks!